docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+- restaurant with `"id": "8"` was created from scratch using the API, passing the `locale: fr` in the request body (see [creating a new localized entry](#creating-a-new-localized-entry)).
+:::
+::::
+
+### Creating a new localized entry
+
+To create a localized entry from scratch, send a POST request to the Content API.
+
+If no locale has been passed in the request body, the entry is created using the default locale for the application:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant B",
+  "slug": "restaurant-b",
+  "stars": 1,
+  "description": "Restaurant with terrace in front of the lake"
+}
+```
+
+:::
+
+::: tab Response
+
+**Example response**
+
+```json
+{
+    "id": 5,
+    "name": "Restaurant B",
+    "slug": "restaurant-b",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant with terraza in front of the lake",
+    "menu": null,
+    "locale": "en",
+    "published_at": "2021-04-07T12:51:43.681Z",
+    "created_at": "2021-04-07T12:51:43.695Z",
+    "updated_at": "2021-04-07T12:51:43.695Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+To create a localized entry for a locale different from the default one, add the `locale` attribute to the body of the POST request:
+
+:::: tabs
+::: tab Request
+
+**Example request**
+
+```js
+POST http://localhost:1337/restaurants
+```
+
+```json
+{
+  "name": "Restaurant F",
+  "slug": "restaurant-f",
+  "stars": 1,
+  "description": "Restaurant face à la mer",
+  "locale": "fr"
+}
+```
+
+:::
+
+::: tab Response
+**Example response**
+
+```json
+{
+    "id": 8,
+    "name": "Restaurant F",
+    "slug": "restaurant-f",
+    "price_range": null,
+    "contact_email": null,
+    "stars": 1,
+    "averagePrice": null,
+    "address": null,
+    "timestamp": null,
+    "short_description": null,
+    "since": null,
+    "description": "Restaurant face à la mer",
+    "menu": null,
+    "locale": "fr",
+    "published_at": "2021-04-07T13:22:46.589Z",
+    "created_at": "2021-04-07T13:22:46.617Z",
+    "updated_at": "2021-04-07T13:22:46.617Z",
+    "closing_period": null,
+    "services": [],
+    "opening_times": [],
+    "dz": [],
+    "cover": null,
+    "images": [],
+    "categories": [],
+    "localizations": []
+}
+```
+
+:::
+::::
+
+### Creating a localization for an existing entry
+
+To create another localization for an existing localized entry, send a POST request to the appropriate URL depending on the type of content:
+
+| Content-Type    | Request URL format                       |
```

Comment:
Holla todos, big question that needs to be clarified in a Guideline document.
Here I can ping @meganelacheny and @maevalienard cause it will/can impact the wording in the admin interface.
- [ ] Content Type
- [ ] Content-Type
- [ ] Content type
- [ ] Content-type
- [ ] content type
- [ ] content-type

Reply:
Already [raised the issue](https://github.com/strapi/documentation/pull/199#issuecomment-802971394) with Mégane's "Content-Type Builder" docs.

To me, Content Type without a dash would be the most appropriate.

Reply:
We already had this discussion @lauriejim when I started the user guide and I thought we all agreed on "content-type"? If I'm not mistaken, it's also the way it's written in the company's glossary on Notion  🤔  It can be changed if eventually that's not the best option though.
By the way, as we're discussing wording: in the user guide we also normally write "single type" and "collection type" (in this doc I've seen it written differently, like "Single Type" or "Collection Type" even in the middle of a sentence which looks a bit odd in my opinion).

Reply:
Indeed, I title-cased Single Type and Collection Type because this is how it's written in most of the dev docs I've seen so far. But maybe it's time to add consistency as well! haha
So I figured out we would title-case concepts that are unique to Strapi 🤷